tests/integration/components/bs-modal-test.js

```diff
@@ -312,5 +312,35 @@ module('Integration | Component | bs-modal', function (hooks) {
       await settled();
       assert.dom('.modal').doesNotExist();
     });
+
+    test('it allows to render modal in fullscreen using @fullscreen argument', async function (assert) {
+      this.set('fullscreen', null);
+      await render(hbs`<BsModal>`);
```

Comment:
You need to close the `<BsModal>` component. Otherwise it is a syntax error, which breaks the build.

```suggestion
      await render(hbs`<BsModal>Hello World!</BsModal>`);
```

Reply:
Of course !! My mistake